templates/index.html

```diff
@@ -30,10 +30,15 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
     Vacation Planner
 </h1>
 <div class="container">
-    <h5 style="color: #24c1e0">
-        Please enter the city, country and weekday you want to visit
-    </h5>
-    <div>
+    <div class="column">
+        <h5 style="color: #24c1e0">
+            Please enter the city, country and weekday you want to visit
+        </h5>
+    </div>
+    <div class="column">
+        <button class="btn btn-outline-info btn-sm" id="find-my-location"> Find my location</button>
```

Comment:
It might make more sense to say "Autofill current Location "  or just `Autofill` ?

Reply:
I think `Autofill` makes sense